README.md

```diff
@@ -1,24 +1,73 @@
-# README
+# MICRO-REDDIT APP
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> A micro version of the reddit web app made with rails
 
-Things you may want to cover:
+A simple platform where users can post links and make comments on them.
 
-* Ruby version
+## Built With
 
-* System dependencies
+- Ruby on Rails version 5.1.6
+- VSCode, Linters, Rubocop
 
-* Configuration
 
-* Database creation
+## Getting Started
 
-* Database initialization
+To get a local copy up and running follow these simple example steps.
 
-* How to run the test suite
+- Clone the repository and there you go! ;-)
 
-* Services (job queues, cache servers, search engines, etc.)
+### Prerequisites
+
+- Get ruby latest version installed
+- Get rails 5.1.6 or later installed
+- Get rubocop installed
+- Get sqlite installed if you are using any other system than Linux
+
+### Setup
+
+- Clone the repository on your local machine
+- cd into the folder
+- install the the bundles with the `bundle` or `bundle install` command. If you are asked to do `bundle update` before, do it.
+- Set up the database using `rake db:migrate` or `rails db:migrate`
+
+### Install
+
+- Install VSCode or any code editor you like
+- Install Ruby on rails 5.1.6 or later if you don't have it yet
+- Run this command on your terminal in order to install rubocop: gem install rubocop 
+
+### Run tests
+
+- In the command line, cd in the project folder
+- Type `rails console` of `rails c` command to get in the console
+- Use the command to create a new user named Test: `User.create(username:'test', email:'test@testland.com', password:'test2020', pass_confirm:'test2020')`
+- Use this command to make 'Test' user create a post: `User.first.create(url:'https://microverse.org', title:'Awesome bootcamp')`
```

Comment:
- [ ] This command is giving this error let's fix it:

![image](https://user-images.githubusercontent.com/52209067/80896131-5237bb80-8ca8-11ea-95db-0c0f6726928b.png)